test/templates/bad/limit_size.yaml

```diff
@@ -1,6 +1,8 @@
 ---
 AWSTemplateFormatVersion: "2010-09-09"
-Description: A sample template that breaks every limit
+Description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. In aliquet justo nibh, a venenatis justo suscipit et. Sed vulputate est non vulputate cursus. In et mollis nunc. Ut semper justo nec odio dignissim semper. Sed interdum elementum ante. Phasellus bibendum mattis ultrices. Nullam varius dui mi, ac fermentum ante sodales nec. Mauris id libero id turpis sollicitudin imperdiet. Praesent volutpat, elit ut malesuada ultricies, magna velit ullamcorper leo, sagittis pulvinar elit erat faucibus felis.
```

Comment:
Bonus points for Lorem ipsum!

Reply:
https://www.lipsum.com/ allowed generating specific byte lengths... ❤️